qml/pages/BookmarkSyncPage.qml

```diff
@@ -0,0 +1,86 @@
+/*
+The MIT License (MIT)
+
+Copyright (c) 2014-2017 Oleg Linkin <maledictusdemagog@gmail.com>
```

Comment:
Use your info because it is your contribution!

Reply:
good call